torch/_inductor/codegen/triton.py

```diff
@@ -1536,6 +1537,12 @@ def indexing(
                 # indirect indexing
                 cse_var = self.cse.varname_map[var.name]
                 mask_vars.update(cse_var.mask_vars)
+            elif "(" in var.name:
```

Comment:
cc @Chillee If I can figure out how to strip out this leading `(` so I can actually match against symT template indexes I think we have good support for all the divisible cases